caravel/assets/javascripts/explorev2/stores/store.js

```diff
@@ -1655,16 +1658,11 @@ export const initialState = {
   metricsOpts: [],
   columnOpts: [],
   orderingOpts: [],
+  searchBox: false,
```

Comment:
I think we could delete 1661-1664 here since we now have them in form_data. I deleted it in a previous PR but somehow it must be accidentally restored during the rebase.